helm/templates/CSIDriver.yaml

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
can we remove the top and bottom empty lines in these manifests? just only empty bottom line would be enough

Reply:
These files are auto generated.  IMO this is OK.